meg_runtime/locking/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""Multimedia Extensible Git File Locking"""
+
+from meg_runtime.locking.manager import *
```

Comment:
Missing a new line at the end of the file.

Reply:
Oh also, only import want you want to be exported from the module, I very much doubt you want to export everything imported into that module.